packages/suite-web-landing/src/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
browser detection is not applied on current landing page and I believe it's correct